example-jobs/SampleBaseJobs.groovy

```diff
@@ -39,6 +39,16 @@ new BaseJobBuilder(
     }
 }
 
+new BaseJobBuilder(
+        name: "sample-base-job-with-performance-publisher",
+        description: "A job with a performance publisher. It does not include the actual bits that run the load tests"
+).build(this).with {
+    steps {
+        shell("echo 'Run jmeter tests here'")
+    }
+    CommonUtils.addPerformancePublisher(delegate,failedThresholdPositive=10, failedThresholdNegative=10, unstableThresholdPositive=5, unstableThresholdNegative=5)
```

Comment:
I went back and forth on that. Having built a number of these load test jobs, I can't think of any common enough settings that would be good defaults.